doc/changes/changes_1.1.0.md

```diff
@@ -0,0 +1,5 @@
+# Test Database Builder 1.1.0, released 2020-XX-XX
```

Comment:
Add summary.

Reply:
I left it out, since I have to merge with the other PR anyway.